content/guides/concepts/dnslink.md

```diff
@@ -43,3 +51,24 @@ Will get you this block:
 ```
 /ipfs/QmYNQJoKGNHTpPxCBPh9KkDpaExgd2duMa3aF6ytMpHdao/media/
 ```
+
+### Publish via Subdomain
+
+It is possible to publish DNSLink using a special subdomain named `_dnslink`.
```

Comment:
Can you describe a use case for this? I didn’t know it existed and I’m wondering why you’d want to do this instead just adding another TXT record directly on the domain you want to use.

Reply:
I've added a paragraph with generic use case. Basically its useful when you want to delegate control over DNSLink without allowing for changes to anything else.

FYI there is another one, but it is specific to a vendor and probably should not land in the doc: Historically we had a bug where [dnsimple](https://dnsimple.com/) automatically added a TXT record "this is an ALIAS record" in some setups, but instead of making a new TXT record they overrode  existing TXT record holding the dnslink, effectively breaking it.  Using subdomain works around that as well.